src/main/kotlin/com/rinha/application/controller/PersonController.kt

```diff
@@ -0,0 +1,49 @@
+package com.rinha.application.controller
+
+import com.rinha.application.dtos.requests.PersonRequest
+import com.rinha.application.dtos.responses.PersonResponse
+import com.rinha.application.dtos.responses.PersonResponse.Companion.toPersonResponse
+import com.rinha.application.dtos.responses.PersonResponse.Companion.toPersonResponseList
+import com.rinha.domain.ports.input.PersonPort
+import java.util.UUID
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("")
+class PersonController(
+    private val personPort: PersonPort
```

Comment:
I was thinking that the controller actually would inherit from `PersonPort` and somehow be injected into the service, but that don't even make sense :sweat_smile: